### DISCUSSION TITLE
Can't install Windows Community Toolkit Data Grid

### DISCUSSION THREAD

**Discussion by n2-holden** [unoplatform/uno#10796] [Q&A]
I get an error from Nuget: microsoft.toolkit.uwp.ui.controls.datagrid is not compatible with net6.0-Windows10.0.19041

I think I'm following the directions, but it seems odd that I would need to install a separate package just for the DataGrid.  Isn't there a single package for the WCT?  When I browse for "windows community toolkit" I get dozens of hits in NuGet, none of which seem correct.

**Comment by jeromelaban** [comment]
Thanks for the report. You can find [documentation on how to install the community toolkit](https://platform.uno/docs/articles/uno-community-toolkit.html) in your app, as well as [a sample about the DataGrid here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/WCTDataTreeTabSample).
